docs/build/program/apis/data-client.md

```diff
@@ -705,3 +756,89 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 
 {{% /tab %}}
 {{< /tabs >}}
+
+### AddBoundingBoxToImageById
+
+Add a bounding box to an image specified by its id.
```

Comment:
```suggestion
Add a bounding box to an image by its specified id.
```
Just a suggestion!

Reply:
Ooh, this is tricky -- original way ties `its id` to `image` a bit more clearly, I worry that the rewrite ties it back to `box` erroneously. Definitely tricky either way.

But these should all be something better than `id` anyways, update everywhere to be `specified by its BinaryID`.

Reply:
This makes a lot of sense, thank you for the explanation